redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/provision/handler/UpdateDatabaseServerRegistrationHandler.java

```diff
@@ -64,6 +65,12 @@ public void accept(Event<UpdateDatabaseServerRegistrationRequest> event) {
             dbServerConfig.setHost(dbHostname.getName());
             dbServerConfig.setPort(Integer.parseInt(dbPort.getName()));
 
+            if (CloudPlatform.valueOf(dbStack.getCloudPlatform()) == CloudPlatform.AZURE) {
```

Comment:
I would put this code to UserGeneratorService, because we already have classes there performing platform specific utilities, like password generation and so, and I think it is great to have these platform specific things collected to one location. Also, those generators I would at some point move into one package (3 files already).

Reply:
Done!